web/components/feed/feed-items.tsx

```diff
@@ -137,7 +145,15 @@ export function FeedComment(props: {
     bought = bet.amount >= 0 ? 'bought' : 'sold'
     money = formatMoney(Math.abs(bet.amount))
   }
-  const { text, userUsername, userName, userAvatarUrl, createdTime } = comment
+  const { text, userUsername, userName, userAvatarUrl, createdTime, userId } =
+    comment
+
+  const userBets = useUserContractBets(userId, contract.id)
```

Comment:
Experimentation is good! But for a contract with lots of comments this could be an n^2 thing.

If it would still work on markets with lots of comments without locking up, this could be fine for a short while, but you'd probably want a comment explaining that.

Reply:
Yes, true! I expected you to read my mind here, next time I'll transmute my thoughts into a comment.